oeis.py

```diff
@@ -375,6 +375,31 @@ def A133058(start: int = 0, limit: int = 20) -> Collection[int]:
     return sequence[start:]
 
 
+@oeis
+def A000121(start: int = 0, limit: int = 20) -> Collection[int]:
+    """Number of representations of n as a sum of Fibonacci
+    numbers (1 is allowed twice as a part).
+    """
+    sequence = []
+    for i in range(0, limit):
+        sequence.append(recursivity(start + i))
+    print(sequence)
+    return sequence
+
+
+def recursivity(n: int, recurseHigh: bool = True) -> int:
+    if n == 0:
```

Comment:
This funciton lacks a docstring and a proper name. You could even "hide" it by putting it inside the A000121 body.